CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hey, welcome to the party! 🎉
+
+Thank you so much for contributing to EasyMDE. 😘
+
+
+## Asking questions, suggesting wonderful ideas or reporting bugs
+
+You can [submit an issue️](https://github.com/Ionaru/easy-markdown-editor/issues/new) on this GitHub repository.
+
+
+## Coding
+
+### 📦 Prerequisites
+
+You need node.js and npm.
+
+To install them on Debian-based systems:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
```

Comment:
Would take the most recent LTS version of node: `https://deb.nodesource.com/setup_10.x`

Reply:
yep, done.